### DISCUSSION TITLE
tiny editor not full

### DISCUSSION THREAD

**Discussion by InfGame** [voidlabs/mosaico#695] [Q&A]
We jst upgraded from 0.18.6 to the newest version because we encountered a bug with the older version and have noticed that some of our older templates don't feature the full tiny editor anymore
talking about this one:
![grafik](https://github.com/voidlabs/mosaico/assets/74739713/a002ce0d-696e-4e1a-9a55-b545f7788bf7)

Instead they all look like this now:
![Screenshot from 2023-08-02 14-02-49](https://github.com/voidlabs/mosaico/assets/74739713/2b2afa4e-d88b-403e-b8fc-7290f4f52872)

is there a setting or something else to activate this type of editor again?

**Comment by bago** [comment]
"Variablen" in your first screen is something that doesn't belong to an official mosaico release.
I guess you used a customized mosaico before your upgrade.

If not, please tell me which official release produces the first result.

**Comment by InfGame** [comment]
we were using the unstable version of mosaico 0.18.6 before upgrading

Edit: if it helps more we used version 0.17.4 before unstable 0.18.6

**Comment by bago** [comment]
"Variablen" is not something that could have been in any release (stable or unstable), AFAIK.

**Comment by InfGame** [comment]
ok. thanks for the help

**Comment by InfGame** [comment]
looks like the "variables" button was a plugin into tiny made by us a longer time ago. was able to recreate it. thank you for your help
